kola-denylist.yaml

```diff
@@ -27,3 +27,9 @@
   tracker: https://github.com/coreos/coreos-assembler/issues/2725
   arches:
   - ppc64le
+- pattern: ext.config.shared.chrony.dhcp-propagation
+  tracker: https://github.com/openshift/os/issues/1352
+  snooze: 2023-09-22
+  arches:
+  - s390x
+  - ppc64le
```

Comment:
Maybe we should add `warn: true` to this snooze as well.

Reply:
Unfortunately we didn't really prioritize backporting that new feature to older branches of COSA so it won't do much. See https://github.com/coreos/coreos-assembler/pull/3581

Reply:
I see, thanks for the info! :+1:

Reply:
Oh didnt know about this new feature.